### DISCUSSION TITLE
Can't publish a meeting after entering certain fields

### DISCUSSION THREAD

**Discussion by dbennett455** [code4recovery/12-step-meeting-list#1202] [Q&A]
We have successfully added a few meetings into the plugin.   At some point the [ Publish ] button became greyed out during editing.  For example,   after entering the Online Meeting URL we can no longer publish the meeting.   There are some errors in the js console:

![image](https://github.com/code4recovery/12-step-meeting-list/assets/185783/20c79cda-ce25-4b28-a75e-45b6d6538a6d)

[camissouri.org-1693161514878.log](https://github.com/code4recovery/12-step-meeting-list/files/12448595/camissouri.org-1693161514878.log)

Once the [ Publish ] button becomes grey,  it never returns to blue even after removing the value that seemingly caused it to turn grey.

**Comment by joshreisner** [comment]
hi @dbennett455 i think what is going on here is your have a google maps api key that is not activated. when i check out your site i see this error:

<img width="1374" alt="Screenshot 2023-08-27 at 12 19 33 PM" src="https://github.com/code4recovery/12-step-meeting-list/assets/1551689/dd0d906a-a0f4-4353-8db5-a1739c308890">

which has this link to fix the issue: https://developers.google.com/maps/documentation/javascript/error-messages#api-not-activated-map-error

we should be doing a better job of surfacing errors on this page though, we have an issue to track that work: https://github.com/code4recovery/12-step-meeting-list/issues/1002

at any rate, try fixing the API key (or switch to mapbox) and let us know if that solves it for you, thanks

**Reply by joshreisner** [reply]
@dbennett455 ok great, thanks for fixing that, and sorry to hear it's still not working. what address are you entering for this meeting? this also affects whether the publish button lights up. 

as a reminder, because it also says it on the page, every meeting needs to have *some* kind of location, even if it's an approximate address, like a city name.

**Reply by joshreisner** [reply]
@dbennett455 we just released version `3.14.21` which should fix the javascript errors you're seeing - not sure if it will fix your issue entirely or not, but please let us know how it goes. you will still need some kind of valid location (street address or city) in the address field

**Comment by dbennett455** [comment]
Hi,

Thanks for your answer.   I was able to get the maps API working and tested with another meeting that had already been entered,  however,   the publish button still greys out as soon as I enter an Online Meeting URL.   The javascript console errors are a little bit different now....

![image](https://github.com/code4recovery/12-step-meeting-list/assets/185783/47070c6c-0ad6-432a-8462-821bbd19fef2)

![image](https://github.com/code4recovery/12-step-meeting-list/assets/185783/5d5c6303-c666-4479-9077-8459ffa6e7b5)

[camissouri.org-1693182335245.log](https://github.com/code4recovery/12-step-meeting-list/files/12449154/camissouri.org-1693182335245.log)
